main/src/main/java/org/mobilitydata/gtfsvalidator/notice/AgencyLangInconsistencyNotice.java

```diff
@@ -23,8 +23,8 @@
  *
  * <p>Severity: {@code SeverityLevel.WARNING}
  */
-public class InconsistentAgencyLangNotice extends ValidationNotice {
-  public InconsistentAgencyLangNotice(long csvRowNumber, String expected, String actual) {
+public class AgencyLangInconsistencyNotice extends ValidationNotice {
```

Comment:
What is the rationale for this change?

Reply:
@barbeau  suggested in https://github.com/MobilityData/gtfs-validator/pull/781#discussion_r585866237 we 

> put the entity name first when it makes sense for consistency, and because if we list the rules in the table alphabetically then, for example, most of the Route rules should be grouped together. 

This change would allow to regroup notices related to `agency.txt`